src/libmongoc/src/mongoc/mongoc-crypt.c

```diff
@@ -696,26 +703,24 @@ _try_add_aws_from_env (bson_t *out, bson_error_t *error)
 }
 
 /**
- * @brief Attempt to load an Azure access token from the environment and append
- * them to the kmsProviders
+ * @brief Attempt to request a new Azure access token from the IMDS HTTP server
  *
- * @param out A kmsProviders object to update
- * @param error An error-out parameter
- * @retval true If there was no error and we loaded credentials
- * @retval false If there was an error obtaining or appending credentials
+ * @param out The token to populate. Must later be destroyed by the caller.
+ * @param error An output parameter to capture any errors
+ * @retval true Upon successfully obtaining and parsing a token
+ * @retval false If any error occurs.
  */
 static bool
-_try_add_azure_from_env (bson_t *out, bson_error_t *error)
+_request_new_azure_mid_token (mcd_azure_access_token *out, bson_error_t *error)
```

Comment:
What does "mid" refer to?

Reply:
"managed identity" in this case. I can change that if it is unclear.

Reply:
Since this is the only reference to "mid", I suggest renaming to `_request_new_azure_access_token` or defining "mid" in a comment.